source/extensions/filters/common/expr/evaluator.cc

```diff
@@ -65,6 +65,11 @@ absl::optional<CelValue> StreamActivation::FindValue(absl::string_view name,
         Protobuf::Arena::Create<FilterStateWrapper>(arena, *arena, info.filterState()));
   case ActivationToken::XDS:
     return CelValue::CreateMap(Protobuf::Arena::Create<XDSWrapper>(arena, *arena, info));
+  case ActivationToken::Node:
```

Comment:
Maybe use "xds.node" as the attribute for the xDS Node?

Reply:
it's ok to use that, but `node` is used in wasm, it's better to be consistent?

Reply:
We already changed some top-level attributes, e.g. `route_metadata` is now `xds.route_metadata`. The same applies here, I think. Wasm can use either way, and we can slowly deprecate the Wasm-specific attributes.